.github/workflows/install_spack.sh

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env sh
-git clone https://github.com/spack/spack.git
-echo -e "config:\n  build_jobs: 2" > spack/etc/spack/config.yaml
-. spack/share/spack/setup-env.sh
-spack compilers
+. share/spack/setup-env.sh
+echo -e "config:\n  build_jobs: 2" > etc/spack/config.yaml
```

Comment:
we could even do:
```suggestion
spack config add config:build_jobs:2
```
now, do we? :)

Reply:
Almost https://github.com/spack/spack/issues/17543

Reply:
Ouch :)